backend/server/models/Company.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const CompanySchema = new mongoose.Schema({
+  name: { type: String, required: true },
+  companyOwner: { type: String, required: true },
+  phoneNumber: { type: String, required: true },
+  city: { type: String, required: true },
+  state: { type: String, required: true },
+  postalCode: { type: String, required: true },
+  logo: { type: String },
+  deals: { type: Schema.Types.ObjectId, ref: "Deal" },
```

Comment:
Should deals be wrapped in an array? 

Like this: deals: [{ type: Schema.Types.ObjectId, ref: "Deal" }]

Reply:
yes. i caught that just now when posting to db!